resources/vue/src/components/account/common/Account.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="container d-flex">
+        <account-aside></account-aside>
+        <account-main></account-main>
+    </div>
+</template>
+
+<script>
+import AccountAside from "./AccountAside";
+import AccountMain from "./AccountMain";
```

Comment:
In general component composition is better organized either with flat lowercased names, or in folders like you did, but without common folder. 
Like so:
account/Account.vue
account/AccountAside.vue
account/AccountMain.vue
Some of the tips about flat structure can be found here: https://www.youtube.com/watch?v=VHVKYDIEtaI